content/en/blog/_posts/2021-07-19-alpha-swap-support.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: blog
+title: 'New in Kubernetes v1.22: alpha support for using swap memory'
+date: 2021-07-19
```

Comment:
This will go out post-release, right?

Reply:
This blog is scheduled to go out on August 25th.

Reply:
There were some minor changes, and the 1.21 comms team decided to move the date to 18th August. @ehashman will update the same here.